simpletransformers/language_representation/transformer_models/bert_model.py

```diff
@@ -0,0 +1,33 @@
+from transformers.modeling_bert import BertModel, BertPreTrainedModel
+
+
+# supports both BERT & ROBERTA BASED MODELS
+class BertForTextRepresentation(BertPreTrainedModel):
```

Comment:
it inherits from `BertPreTrainedModel ` to follow the pattern in: https://github.com/pablonm3/simpletransformers/blob/d36f9327369bb36a93c8cedfbe08e8900788acf0/simpletransformers/classification/transformer_models/bert_model.py#L7 I still don't get what are the reasons for doing this inheritance?

Reply:
The `BertPreTrainedModel` itself inherits from `PreTrainedModel`. This means that a bunch of methods for things like loading, saving, and other configurations gets inherited. Also, it helps to keep all Simple Transformers custom models tied to the implementations in Hugging Face.

_I haven't had time to test this PR yet, but it looks good from a quick skim. Thank you!_